snap/snapcraft.yaml

```diff
@@ -166,14 +179,35 @@ parts:
       - gstreamer1.0-libav
       - dbus-x11
       - projectm-data
+      - qtwayland5 # Attempt at Wayland Support
+      - libqtcore4 # Attempt to get Spotify to work
+      - glibc-source # Attempt to get Spotify to work
+      - plasma-integration
+      - kde-style-breeze
+      - qt5-gtk-platformtheme
+
+plugs:
+  gtk-3-themes:
+    interface: content
+    target: $SNAP/data-dir/themes
+    default-provider: gtk-common-themes
+  icon-themes:
+    interface: content
+    target: $SNAP/data-dir/icons
+    default-provider: gtk-common-themes
+  sound-themes:
+    interface: content
+    target: $SNAP/data-dir/sounds
+    default-provider: gtk-common-themes
 
 apps:
   clementine:
-    command: desktop-launch $SNAP/usr/bin/clementine
+    command: bin/desktop-launch $SNAP/usr/bin/clementine
     desktop: usr/share/applications/clementine.desktop
     environment:
       ALSA_CONFIG_PATH: /snap/$SNAPCRAFT_PROJECT_NAME/current/usr/share/alsa/alsa.conf
       LD_LIBRARY_PATH: $LD_LIBRARY_PATH:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pulseaudio
+      DISABLE_WAYLAND: 1
```

Comment:
This seems counter to the attempt at wayland support above.

Reply:
Needs to stay.

Reply:
I think this is needed for clementine to run as an XWayland app when in a wayland session. Without this environment variable, clementine looks like this in Wayland: 
![Screenshot from 2020-10-06 19-42-12](https://user-images.githubusercontent.com/24817147/95246671-b8eb7300-080c-11eb-91d9-7f3b6f358b48.png)
With the environment variable set in Wayland (running on XWayland) :
![Screenshot from 2020-10-06 19-44-48](https://user-images.githubusercontent.com/24817147/95246711-c9035280-080c-11eb-9a11-b603388a9d8a.png)
As you can see the XWayland app keeps the GTK theming and cursor theme.